hist/histpainter/src/THistPainter.cxx

```diff
@@ -2604,7 +2604,7 @@ Begin_Macro(source)
       z = x*x + y*y;
       h3scat->Fill(x,y,z);
    }
-   h3scat->Draw();
+   h3scat->Draw("SCAT");
```

Comment:
```suggestion
   h3scat->Draw("SCAT"); // This a legacy draw option. Please consider using TScatter instead
```

Reply:
I will add this comment and in other examples too, to be consistent. Not that in that particular example TCatter cannot be used because this a a 3D plot.